src/main/java/ru/curs/bass/App.java

```diff
@@ -75,26 +79,36 @@ public static void main(String[] args) {
                 }
                 File propertiesFile = new File(propertiesPath);
                 if (!(propertiesFile.exists() && propertiesFile.canRead())) {
-                    ch.error(String.format("Properties file %s does not exists or cannot be read.%n",
+                    consoleHelper.error(String.format("Properties file %s does not exists or cannot be read.%n",
                             propertiesFile.getAbsolutePath()));
-                    ch.info(HELP);
+                    consoleHelper.info(HELP);
                     return;
                 }
                 AppProperties properties = readProperties(propertiesFile);
-                properties.setTask(Task.getByString(task));
+                properties.setCommand(Command.getByString(cmd));
+
 
                 try {
-                    Bass bass = new Bass(properties, ch);
-                    bassConsumer.accept(bass);
-                    bass.close();
+                    if (properties.getCommand() == Command.VALIDATE) {
```

Comment:
This code is not clean enough. We can create bass object without initialization of score and connection pool. 

And this block might look like
```
  Bass bass = new Bass(properties, consoleHelper);
  bass.parseScore();
  bassConsumer.accept(bass);
  bass.close();
```